compiler-core/src/parse/error.rs

```diff
@@ -157,6 +157,13 @@ utf16_codepoint, utf32_codepoint, signed, unsigned, big, little, native, size, u
                     "See: https://tour.gleam.run/basics/lists/".into(),
                 ],
             ),
+            ParseErrorType::ListSpreadWithAnotherSpread => (
+                "I wasn't expecting a spread here",
+                vec![
+                    "Hint: use list.append(list1, list2) to join two lists.".into(),
```

Comment:
Not sure we want to recommend this, it's normally the wrong thing to do

Reply:
Yea, I guess it is likely that the problem is modeled poorly. Perhaps we can add that as a comment, or suggest a common "pattern" to model such problems better.

On the other hand, you could argue that it is still helpful for newcomers. You have to type `[..xs, ..ys]` to get the recommendation, so I think it is pretty clearly what you are trying to do.

What do you think?

Reply:
What do you think of the new message?